client/src/components/Card.css

```diff
@@ -1,27 +1,88 @@
+.body {
+  justify-content: center;
+  align-items: center;
+}
+ 
 .card {
   margin: 20px;
   padding: 20px;
   background-color: white;
   border-radius: 20px;
   display: flex;
   flex-direction: column;
-  justify-content: space-around;
+  justify-content: center;
   align-items: center;
   width: 400px;
 }
 
 img {
   width: 100%;
-  height: 100%;
+  height: 100px;
   object-fit: contain;
   overflow: hidden;
+  margin: 25px;
+}
+
+
+p.description {
+  margin: 40px;
+  width: 100%;
+  height: 300px;
+  font-size: 25px;
+  /* border: 3px solid green;  */
+}
+
+.detailsButton {
+ background-color: #66BFBF;
+ height: 50px;
+ width: 100px;
+ border-radius: 10px;
+ font-size: 15px;
+ color: white;
+ border-color: 1px solid #66a7bf
 }
```

Comment:
nit: it appears the indent is not consistent here with the rest of the file

Reply:
thanks, fixed it!